ddclient

```diff
@@ -4816,7 +4816,7 @@ sub nic_duckdns_update {
 
         # Set the URL that we're going to to update
         my $url;
-        $url  = "http://$config{$h}{'server'}/update";
+        $url  = "https://$config{$h}{'server'}/update";
```

Comment:
bc1qex7a52dr8dz9dt7sn76t5fu9pdyzluwsdawapc

Reply:
not sure what this means?!